Yona/Yona/Others/SMSValidPasscodeLogin/Login/LoginViewController.swift

```diff
@@ -9,15 +9,19 @@
 import UIKit
 import LocalAuthentication
 
-class LoginViewController: LoginSignupValidationMasterView, AppLifeCylcleConsumer {
-    var loginAttempts:Int = 1
-    fileprivate var totalAttempts : Int = 3
+class LoginViewController: LoginSignupValidationMasterView {
+    
     @IBOutlet weak var bottomSpaceContraint: NSLayoutConstraint!
     @IBOutlet var pinResetButton: UIButton!
     @IBOutlet var closeButton: UIBarButtonItem?
     @IBOutlet var accountBlockedTitle: UILabel?
+    
+    var loginAttempts:Int = 1
+    var totalAttempts : Int = 3
```

Comment:
This is the maximum, right? Then name like that: ``maxAttempts``.

Reply:
Yes. Updated.